language/lexer/lexer.go

```diff
@@ -237,6 +240,7 @@ func readString(s *source.Source, start int) (Token, error) {
 					valueBuffer.WriteRune('"')
 					break
 				case '/':
+					valueBuffer.WriteRune('\\')
```

Comment:
Hmm, this was earlier added to satisfy the test for `TestLexer_LexesStrings`

```
		{
			Body: "\"slashes \\\\ \\/\"",
			Expected: Token{
				Kind:  TokenKind[STRING],
				Start: 0,
				End:   15,
				Value: "slashes \\ \\/",
			},
		},


```

(previously ```valueBuffer ...``` was ```value += "\\/"```)

Turns out the test was wrong, it should be:
```
		{
			Body: "\"slashes \\\\ \\/\"",
			Expected: Token{
				Kind:  TokenKind[STRING],
				Start: 0,
				End:   15,
				Value: "slashes \\ /",
			},
		},
```
(Reference: https://github.com/graphql/graphql-js/blob/master/src/language/__tests__/lexer-test.js#L142)

I'll make the changes and push it back to your branch, nice catch! 👍